css-three-column-layout/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div class="container">
+    <div class="row">
+      <h1> Designed for life science. Optimized for SaMD and Digitial Health.</h1>
+    </div>
+    <div class="row">
+      <div class="image-container">
+        <img src="images/merge.png" alt="merge">
+      </div>
+      <div class="image-container">
+        <img src="images/cloud-computing.png" alt="cloud-computing">
+      </div>
+      <div class="image-container">
+        <img src="images/discussion.png" alt="discussion">
+      </div>
+    </div>
+    <div class="row">
+      <div class="description">
+        <h2> Streamlined QMS</h2>
+        <p> A QMS that includes modules for all stages of the product life cycle from Design Control to CAPA </p>
+      </div>
+      <div class="description">
+        <h2 class="too-small"> Integrates with existing tools</h2>
+        <p> Enzyme QMS integrates with your existing tools, such as JIRA, GitHub. and Jenkins with Enzyme </p>
+      </div>
+      <div class="description">
+        <h2> In-depth Onboarding </h2>
+        <p> We train your entire team and provide you with support when and as it's needed</p>
+      </div>
+    </div>
```

Comment:
Try reorganizing your HTML so that there is one row of 3 columns, each with an image, heading, and paragraph in it.

You can make perfect circles by making your image container a square and then setting its border radius to 50%

Reply:
![image](https://user-images.githubusercontent.com/109745413/186781939-09add378-747c-4618-9756-32852c10f1cf.png)